### PR TITLE
Update team allocations for multi-team organisations

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Hopefully these match what's in Discord?

Numbers are based on https://docs.google.com/spreadsheets/d/12Z32_O32DnFiDHX1jqOSQyMVi_g8ElGn6328Nx80o2g/edit.

### CODE REVIEW

File: teams.yaml [hunk 1]
```diff
@@ -5,23 +5,44 @@ teams:
...
-    HAB:
-        name: Haberdashers' Aske's Boys' School
+    HAB1:
+        name: Haberdashers' Aske's Boys' School (Team 1)
+        rookie: No
+    HAB2:
+        name: Haberdashers' Aske's Boys' School (Team 2)
+        rookie: No
+    HRS1:
+        name: HIlls Road SIxth Form College (Team 1)
```
Comment (sedders123) [inline teams.yaml]: Is the casing here intentional?

**Review by sedders123 — COMMENTED** [review]
The teams here match the teams in discord. (There are two additional teams WAL1 and WAL2 in discord but I believe these are not actual teams)

File: teams.yaml (hunk 1 above)
Comment (PeterJCLaw) [reply teams.yaml]: I don't think so. It's all copy/pasted from the sheet.

Comment (PeterJCLaw) [reply teams.yaml]: Fixed here (04b2f24) and in the sheets.

Comment (sedders123) [reply teams.yaml]: 👍 Thanks. Looks like `SIxth` is still odd

Comment (PeterJCLaw) [reply teams.yaml]: And again: 2af70ce; :facepalm: (also fixed the sheets)

**Review by sedders123 — APPROVED** [review]

**Pull request merged**